metadata-jobs/mae-consumer-job/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     avro project(path: ':metadata-models', configuration: 'avroSchema')
 
     compile project(':li-utils')
+    compile project(':gms:client')
```

Comment:
Once my merge PR gets in we may need to adjust this

Reply:
circular dependency?